DependencyInjection/PayumExtension.php

```diff
@@ -38,7 +39,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         // load services
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('payum.xml');
+        $loader->load(Kernel::MAJOR_VERSION === 3 ? 'payum3x.xml' : 'payum.xml');
```

Comment:
Just pushed, it works but still show the deprecation warnings. It's the `XmlFileLoader` that throws them when xml files are read. Should I stay to the previous version ?